### PR TITLE
[Stack HCI] upgrade api-version to 2020-10-01

### PR DESCRIPTION
**Pull request by zhoxing-ms**
Upgrade api-version to 2020-10-01 and bump `vendored_sdks`.
Some breaking changes of SDK are adapted

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Stack HCI

**Comment by yonzhan** [comment]
This RP will be shipped by S178 release.

File: src/stack-hci/azext_stack_hci/generated/utils.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+from importlib import import_module
+
+'''
+Since the method get_models()/get_sdk() in azure-cli-core is bound to resourceType, the method _sdk_get_versioned_sdk() will be called, so only the multi-api SDK is supported.
+If we add a version directory to the path of the SDK folder, because there is no required client (specific to the multiple version SDK) outside of the version directory, the client_factory will have a load problem.
+So write a separate method to load the model.
+'''
+
+
+def get_model(sdk_path, mod_attr_path, checked=True):
```
Comment (Juliehzl) [inline src/stack-hci/azext_stack_hci/generated/utils.py:14]: seems we could import model directly. If yes, no need to add this function

Comment (zhoxing-ms) [reply src/stack-hci/azext_stack_hci/generated/utils.py:14]: Yes, I'll import it directly~

**Review by Juliehzl — APPROVED** [review]

**Pull request merged**